tests/unit_tests/test_game.py

```diff
@@ -72,6 +72,29 @@ def test_instance_logging(game_stats_service):
     await game.on_game_end()
     assert game.validity == ValidityState.FFA_NOT_RANKED
 
+async def test_multi_team(game):
```

Comment:
this test is about invalidity, add that to the method name for easy understandability, see below `uneven_games_not_ranked`, includes what you expect the test to result